package.json

```diff
@@ -1,24 +1,16 @@
 {
   "name": "prismarine-web-client",
-  "version": "1.5.0",
+  "version": "0.0.0-dev",
   "description": "A minecraft client running in a browser",
```

Comment:
Will restore version (I’m just using auto release workflow) tho need to think what to do with workflows files in this or first. The same for main property